sunpy/map/tests/test_mapbase.py

```diff
@@ -27,6 +28,9 @@
 
 testpath = sunpy.data.test.rootdir
 
+@pytest.fixture
+def HMI_file():
```

Comment:
Can you rename this?

Reply:
Rename HMI_file to hmi_file?

Reply:
No, to something similar as the other pytest fixtures in the file.

Reply:
Okay, `hmi_test_map`  will do?